app/scripts/utils/use-click-outside.ts

```diff
@@ -0,0 +1,15 @@
+import { useEffect, useRef } from 'react';
```

Comment:
Not very important, but maybe we can move both use-fix-hooks in the same directory? app/scripts/components/common/page-header/use-mobile-menu-fix.ts is the one i implemented for the resize bug. I don't care which directory, but I like them together :slightly_smiling_face:

Reply:
@AliceR I think the new hook is generic enough to be useful in other places not limited to this component, so it might make sense to keep it in the general utils directory?